### PR TITLE
Translate extending-index with machine translation

### PR DESCRIPTION
**Pull request by egeakman**
Translated extending/index.po with [machine translation](https://github.com/python/python-docs-tr/blob/dddb1c28e96c4b1a9621f451b109d4318b871608/scripts/translate.py)

This is a test case for the new machine translation script. Human review is always required for machine translations.

### CODE REVIEW

File: extending/index.po [hunk 1]
```diff
@@ -30,68 +32,115 @@ msgid ""
...
 
 #: extending/index.rst:40
+#, fuzzy
 msgid ""
-"`Python Packaging User Guide: Binary Extensions <https://packaging.python."
-"org/guides/packaging-binary-extensions/>`_"
+"`Python Packaging User Guide: Binary Extensions "
+"<https://packaging.python.org/guides/packaging-binary-extensions/>`_"
 msgstr ""
+"`Python Packaging User Guide: Binary Extensions "
```
Comment (MorphaxTheDeveloper) [inline extending/index.po:96]: "`Python Paketleme Kullanıcı Kılavuzu: İkili Uzantıları "

**Review by MorphaxTheDeveloper — COMMENTED** [review]
nice

File: extending/index.po (hunk 1 above)
Comment (MorphaxTheDeveloper) [inline extending/index.po:96]: "`Python Paketleme Kullanıcı Kılavuzu: İkili Uzantılar "
```suggestion
"`Python Packaging User Guide: Binary Extensions "
```

**Review by MorphaxTheDeveloper — COMMENTED** [review]
try

File: extending/index.po (hunk 1 above)
Comment (MorphaxTheDeveloper) [inline extending/index.po:96]: "`Python Paketleme Kullanıcı Kılavuzu: İkili Uzantılar "
```suggestion
"`Python Paketleme Kullanıcı Kılavuzu: İkili Uzantılar "
```

**Review by MorphaxTheDeveloper — COMMENTED** [review]
ok.

File: extending/index.po [hunk 2]
```diff
@@ -30,68 +32,115 @@ msgid ""
 "extension modules so that they can be loaded dynamically (at run time) into "
 "the interpreter, if the underlying operating system supports this feature."
 msgstr ""
+"Bu belge, Python yorumlayıcısını yeni modüllerle genişletmek için C veya "
+"C++'da modüllerin nasıl yazılacağını açıklar. Bu modüller sadece yeni "
+"fonksiyonları değil, aynı zamanda yeni nesne tiplerini ve metotlarını da "
+"tanımlayabilir. Belge ayrıca Python yorumlayıcısının bir uzantı dili olarak "
+"kullanılmak üzere başka bir uygulamaya nasıl yerleştirileceğini açıklar. Son"
```
Comment (Ardasak) [inline extending/index.po:39]:
```suggestion
"kullanılmak üzere başka bir uygulamaya nasıl yerleştirileceğini de açıklar. Son"
```

File: extending/index.po [hunk 3]
```diff
@@ -30,68 +32,115 @@ msgid ""
...
 msgid ""
 "This document assumes basic knowledge about Python.  For an informal "
 "introduction to the language, see :ref:`tutorial-index`.  :ref:`reference-"
-"index` gives a more formal definition of the language.  :ref:`library-index` "
-"documents the existing object types, functions and modules (both built-in "
+"index` gives a more formal definition of the language.  :ref:`library-index`"
+" documents the existing object types, functions and modules (both built-in "
 "and written in Python) that give the language its wide application range."
 msgstr ""
+"Bu belge, Python hakkında temel bilgileri varsayar. Dile gayri resmi bir "
```
Comment (Ardasak) [inline extending/index.po:53]:
```suggestion
"Bu belge, Python hakkında temel bilgileri ele alır Dile gayri resmi bir "
```

File: extending/index.po [hunk 4]
```diff
@@ -30,68 +32,115 @@ msgid ""
...
-"more sophisticated approaches to creating C and C++ extensions for Python."
+"part of this version of CPython. Third party tools like `Cython "
+"<https://cython.org/>`_, `cffi <https://cffi.readthedocs.io>`_, `SWIG "
+"<https://www.swig.org>`_ and `Numba <https://numba.pydata.org/>`_ offer both"
+" simpler and more sophisticated approaches to creating C and C++ extensions "
+"for Python."
 msgstr ""
+"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
+"uzantıları oluşturmak için temel araçları kapsar. `Numba "
+"<https://numba.pydata.org/>`_, `Numba <https://numba.pydata.org/>`_, `Numba "
```
Comment (Ardasak) [inline extending/index.po:85]:
```suggestion
"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
"uzantıları oluşturmak için temel araçları kapsar. `Cython "
"<https://cython.org/>`_, `cffi <https://cffi.readthedocs.io>`_, `SWIG <https://www."
"swig.org>`_ ve `Numba <https://numba.pydata.org/>`_ gibi "
"üçüncü taraf araçlar, Python için C ve C++ uzantıları oluşturmaya yönelik "
"hem daha basit hem de daha karmaşık yaklaşımlar sunar."
```

**Review by Ardasak — CHANGES_REQUESTED** [review]
After those changes, LGTM!

File: extending/index.po [hunk 5]
```diff
@@ -30,68 +32,120 @@ msgid ""
...
+"uzantıları oluşturmak için temel araçları kapsar. `Numba "
+"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
+"uzantıları oluşturmak için temel araçları kapsar. `Cython "
+"<https://cython.org/>`_, `cffi <https://cffi.readthedocs.io>`_, `SWIG <https://www."
+"swig.org>`_ ve `Numba <https://numba.pydata.org/>`_ gibi "
+"üçüncü taraf araçlar, Python için C ve C++ uzantıları oluşturmaya yönelik "
+"hem daha basit hem de daha karmaşık yaklaşımlar sunar."
+"<https://numba.pydata.org/>`_ ve `Numba <https://numba.pydata.org/>`_ gibi "
+"üçüncü taraf araçlar, Python için C ve C++ uzantıları oluşturmaya yönelik "
+"hem daha basit hem de daha karmaşık yaklaşımlar sunar."
```
Comment (Ardasak) [inline extending/index.po:93]:
```suggestion
"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
"uzantıları oluşturmak için temel araçları kapsar. `Numba "
"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
"uzantıları oluşturmak için temel araçları kapsar. `Cython "
"<https://cython.org/>`_, `cffi <https://cffi.readthedocs.io>`_, `SWIG <https://www."
"swig.org>`_ ve `Numba <https://numba.pydata.org/>`_ gibi "
"üçüncü taraf araçlar, Python için C ve C++ uzantıları oluşturmaya yönelik "
"hem daha basit hem de daha karmaşık yaklaşımlar sunar."
```

**Review by Ardasak — CHANGES_REQUESTED** [review]
Oops

File: extending/index.po [hunk 6]
```diff
@@ -30,68 +32,117 @@ msgid ""
...
+"for Python."
 msgstr ""
+"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
+"uzantıları oluşturmak için temel araçları kapsar. `Numba "
+"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
+"uzantıları oluşturmak için temel araçları kapsar. `Cython "
+"<https://cython.org/>`_, `cffi <https://cffi.readthedocs.io>`_, `SWIG <https://www."
+"swig.org>`_ ve `Numba <https://numba.pydata.org/>`_ gibi "
+"üçüncü taraf araçlar, Python için C ve C++ uzantıları oluşturmaya yönelik "
+"hem daha basit hem de daha karmaşık yaklaşımlar sunar."
```
Comment (Ardasak) [inline extending/index.po:90]:
```suggestion
"Bu kılavuz, yalnızca CPython'un bu sürümünün bir parçası olarak sağlanan "
"uzantıları oluşturmak için temel araçları kapsar. `Cython "
"<https://cython.org/>`_, `cffi <https://cffi.readthedocs.io>`_, `SWIG <https://www."
"swig.org>`_ ve `Numba <https://numba.pydata.org/>`_ gibi "
"üçüncü taraf araçlar, Python için C ve C++ uzantıları oluşturmaya yönelik "
"hem daha basit hem de daha karmaşık yaklaşımlar sunar."
```

**Review by Ardasak — CHANGES_REQUESTED** [review]
oops again

**Review by Ardasak — APPROVED** [review]

File: extending/index.po [hunk 7]
```diff
@@ -30,68 +32,115 @@ msgid ""
...
 msgid ""
 "This document assumes basic knowledge about Python.  For an informal "
 "introduction to the language, see :ref:`tutorial-index`.  :ref:`reference-"
-"index` gives a more formal definition of the language.  :ref:`library-index` "
-"documents the existing object types, functions and modules (both built-in "
+"index` gives a more formal definition of the language.  :ref:`library-index`"
+" documents the existing object types, functions and modules (both built-in "
 "and written in Python) that give the language its wide application range."
 msgstr ""
+"Bu belge, Python hakkında temel bilgileri ele alır Dile gayri resmi bir "
```
Comment (egeakman) [inline extending/index.po:53]:
```suggestion
"Bu belge, Python hakkında temel bilgiye sahip olduğunuzu varsayar. Dile gayri resmi bir "
```

**Pull request merged**